promgen/templates/promgen/rule_block.html

```diff
@@ -11,7 +11,7 @@
     {% endif %}
     <ul>
       {% for k,v in rule.labels.items|dictsort:0 %}
-      <li class="label label-primary">{{k}}:{{v}}</li>
+      <li class="label label-primary"><code v-pre>{{k}}:{{v}}</pre></code></li>
```

Comment:
Can you give me a screenshot of where the problem is? I'm having difficulty understanding from your example and PR.

I didn't think it was normally possible to use `{{$instance}}` in label values (only in annotations) but I could be misunderstanding something

I believe there is also a syntax error here because you have an extra `</pre>`

It may be possible to handle the same thing with just the existing `<li>` tag doing something like `<li v-pre class="label label-primary">` but I would like a screenshot of where the error is actually happing to better understand the correct fix